src/components/header/generate-modal-options.util.tsx

```diff
@@ -0,0 +1,14 @@
+import { StackNavigationOptions } from '@react-navigation/stack';
+import React, { FC } from 'react';
+
+import { HeaderCloseOptionsButton } from './header-close-options-button/header-close-options-button';
+
+export const generateModalOptions = (
```

Comment:
let's rename this function as it's used nor for modals.  mb `generateIntegratedAppOptions`